packages/hadron-build/lib/target.js

```diff
@@ -762,6 +774,41 @@ class Target {
 
     return res[0];
   }
+
+  static getAssetsForVersion(dir, version) {
+    const configs = supportedDistributions
+      .map((distribution) => {
+        return supportedPlatforms.map((platformConfig) => {
+          return { ...platformConfig, distribution };
+        });
+      })
+      .flat();
```

Comment:
While this works just fine, any reason not to use `.flatMap()` in these places?

Reply:
That's a great suggestion! I just need to build some muscle memory to start using it, that's the only reason I'm not using it here, will update 😄

Reply:
Updated, much cleaner now, ty!